### ISSUE TITLE
FEZ (Finite Element Zurich)

### ISSUE THREAD

**Issue by evaherbst** [open-life-science/ols-2#23]
Project Lead: Eva C. Herbst  (evaherbst) and Dylan Bastiaans (github name here)

Mentor: hdinkel

Welcome to OLS-2! This issue will be used to track your project and progress during the program. Please use this checklist over the next few weeks as you start Open Life Science program :tada:.

***

Week 1 (31 August - 4 September 2020): Meet your mentor!
- [x] Create an account on [GitHub](https://github.com)
- [x] Check if you have access to the HackMD notes set up for your meetings with your mentor
- [x] Prepare to meet your mentor(s) by completing a short homework provided in the HackMD notes
- [x] Complete **your own copy** of the [open leadership self-assessment](https://docs.google.com/document/d/1oQgdfj4lPnypAyb9_Ba0Zt7E8J5L6qMvuKwu0wgQsjs/edit?usp=sharing) and share it to your mentor 
  If you're a group, each teammate should complete this assessment individually. This is here to help you set your own personal goals during the program. No need to share your results, but be ready to share your thoughts with your mentor.
- [x] Make sure you know when and how you'll be meeting with your mentor.

Before Week 2 (7 - 11 September 2020): Cohort Call (Welcome to Open Life Science!)
- [x] Create an issue on the [OLS-2 GitHub repository](https://github.com/open-life-science/ols-2/issues/new) for your OLS work and share the link to your mentor.
- [ ] Draft a brief vision statement using your goals

  [This lesson](https://mozilla.github.io/open-leadership-training-series/articles/introduction-to-open-leadership/stating-your-project-vision/) from the Open Leadership Training Series (OLTS) might be helpful

- [ ] Leave a comment on this issue with your draft vision statement & be ready to share this on the call
- [x] Check the [Syllabus](https://openlifesci.org/ols-2) for notes and connection info for all the cohort calls.

Before Week 3 (14 - 18 September 2020): Meet your mentor!
- [ ] Look up two other projects and comment on their issues with feedback on their vision statement
- [ ] Complete this [compare and contrast assignment](https://docs.google.com/document/d/1ukvqDRIYfvCapVMdE5hWP-0MkLNJ9T65X43O7F336Ac/edit?usp=sharing) about current and desired community interactions and value exchanges
- [ ] Complete your Open Canvas ([instructions](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/), [canvas](https://docs.google.com/presentation/d/1MeJo0TyuMg_waLk1J4q9y1aAqKNMuRBlnmxEChSz-cQ/edit?usp=sharing))
- [ ] Share a link to your Open Canvas in your GitHub issue
- [ ] Start your [Roadmap](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/start-your-project-roadmap/)
- [ ] Comment on your issue with your draft Roadmap
- [ ] Suggest a cohort name at the bottom of the shared notes and vote on your favorite with a +1

Before Week 4 (21 - 25 September 2020): Cohort Call (Tooling and roadmapping for Open projects)
- [ ] Look up two other projects and comment on their issues with feedback on their open canvas.

Week 5 and later
- [ ] Create a GitHub repository for your project
- [ ] Add the link to your repository in your issue
- [ ] Use your canvas to [start writing a `README.md` file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/), or landing page, for your project
- [ ] Link to your README in a comment on this issue
- [ ] Add an [open license](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/sharing-your-work-in-the-open/) to your repository as a file called `LICENSE.md`
- [ ] Add a [Code of Conduct](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-a-code-of-conduct/) to your repository as a file called `CODE_OF_CONDUCT.md`
- [ ] Invite new contributors to into your work!

This issue is here to help you keep track of work as you start Open Life Science program. Please refer to the [OLS-2 Syllabus](https://openlifesci.org/ols-2) for more detailed weekly notes and assignments past week 4.

**Comment by evaherbst** [comment]
**Vision Statement**

Our vision is to create a virtual hub to share modeling workflows for finite element analysis (a modeling method used in biomechanics research). Our website will serve as a place to learn methods (including open access alternatives to commercial software) and form collaborations. The main goal is to form a finite element analysis community across various institutions and disciplines, and to increase access to this modeling methodology.

**Comment by evaherbst** [comment]
Here is our Open Canvas: https://docs.google.com/presentation/d/1C5i3prvOBjZ4K0UCuaWO12oQj_k_RY-gvIivTqCKMJ8/edit?usp=sharing

**Comment by joyceykao** [comment]
@evaherbst I saw on your Roadmap that you're planning a March workshop/mini-conference. Is this something OILS can help with (advertisements, online logistics/fundraising consult, etc.)? Let's talk! :)

**Comment by evaherbst** [comment]
@joyceykao yes please, that would be amazing!! I'll send you a message via Slack :)

**Comment by baileythegreen** [comment]
@evaherbst What form is the data used in Finite Element Analysis in? I'm wondering how the field compares to genetics, where a lot of work is on a very large scale, and done on supercomputers and remote servers. There is often a great need for being able to convert files from the output format of one software into the input format of another. If this is also the case in FEA, maybe in addition to recommended workflows, you could have code for doing those things.

Or, all of that could be completely irrelevant. I don't know much about biomechanics :)

**Comment by evaherbst** [comment]
@baileythegreen thanks for your ideas! So there are many components, from unclean meshes to cleaned surface meshes to 3D tetrahedral meshes with node information (which are used for the actual analysis), and then the information about where the forces are applied etc. The setup itself, with the constraints, for running the model can be in various formats depending on the software the researcher is using.

It's a good point about the format conversion. I think most programs have various options (for example reading both .obj and .stl surface meshes) but that is not always the case. I don't think I have time to write any code to convert such formats but I like your idea and will think about including some useful links (and maybe also a chart of which software is compatible with which mesh formats)

Thanks! :)

**Comment by evaherbst** [comment]
The Up-Goer Five:
I found this really difficult since words such as model and shape aren't even included. So the statement isn't specific at all.

We want to make a group to share step by step directions for our type work (using computers to learn how animals work). This will be a place to make it easier for people to learn how to do this and form relationships and work together.

**Comment by Dylan-Bastiaans** [comment]
The Up-Goer Five (I've altered it slightly from your version @evaherbst ):

Our plan is to make a group to share step by step directions to use the computer to study how animals work and to learn about the relationship between head form and bite force. This will be a place to make it easier for people to learn how to approach this and come to talk and work together. In the end we want to interest people from all study directions to come together and add know-how while at the same time they learn themselves new things.

**Comment by malvikasharan** [comment]
@all-contributors please use @evaherbst for idea and content.

**Comment by malvikasharan** [comment]
@all-contributors please add @evaherbst for idea and content.

**Comment by malvikasharan** [comment]
@all-contributors please add @Dylan-Bastiaans for idea and content.

**Comment by LauraCarter** [comment]
The Roadmap for this project looks great - really clear and easy to understand! It might be good to ask possible contributors what kind of community they would like to be part of as well - hopefully this will help you shape things like Slack channels!
